BASE/Test/ServerTelemetryChannel.Test/TelemetryChannel.Tests/TelemetryChannel.Tests.csproj

```diff
@@ -34,6 +34,9 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
     <PackageReference Include="Azure.Core" Version="1.14.0" />
+    
+    <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="2.2.0" />
```

Comment:
suggest using the 2.1*

Reply:
why 2.1 here as opposed to latest?
I have no problem with the downgrade, just trying to understand :)

Reply:
if its testing, yes you can use latest.